packages/odyssey-react/src/components/TextInput/TextInput.tsx

```diff
@@ -164,13 +176,20 @@ export const TextInput = withTheme(
       />
     );
 
-    const search = (
-      <span className={styles.outer}>
+    const faux = (
```

Comment:
This `faux` was simply for my own testing. This would likely become the base for the component, with variants like Password or Search populating the base with specific values (e.g. Search is a TextInput with a SearchIcon prefix and the "search" `type`).